ag-grid-angular-ngcc/src/app/app.module.ts

```diff
@@ -1,7 +1,7 @@
 import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
 
-import 'ag-grid-angular';
+import {AgGridModule} from 'ag-grid-angular';
```

Comment:
@petebacondarwin isn't just a side-effect import enough to trigger ngcc?

Reply:
A side effect import *is* enough to trigger ngcc. But just getting ngcc to run without an error is only half the story. There have been a number of cases where it doesn't error but does produce invalid output. Leading to false positives.

By importing an NgModule it triggers the ngtsc compilation to try to inspect the output and that adds a further check of the generated typings for the library.

But... that is still not really enough because it doesn't catch errors in the JS. To find those we need to add at least one e2e test that runs the app and tries to use a component/directive/pipe from the library.

Reply:
Because of the possibility of a runtime error I added the e2e test instructions in the readme. Makes sense to add an import to the `NgModule` as a pre-requirement 👍